Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
```

Comment:
Wouldn’t it be both safer and more efficient to use the official Tomcat image as a base, rather than a manually maintained image based on a different base operating system (CentOS) than the one official images use (Debian)? I think we could get the size of the stack down from 3 GiB to about half of that, and then even share the base OS with the MySQL container. Moreover, this is a requirement if we aim to make it an official image one day: https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#from